.github/workflows/publish-webbundle.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       run: |
         # Go has been installed at the setup hugo step,
         # you can run setup-go action if you want to install specific version
-        go install github.com/WICG/webpackage/go/bundle/cmd/gen-bundle
+        go install github.com/WICG/webpackage/go/bundle/cmd/gen-bundle@latest
```

Comment:
https://github.com/openameba/a11y-guidelines/actions/runs/6503184992/job/17663276840#step:5:12
> go: 'go install' requires a version when current directory is not in a module
	Try 'go install github.com/WICG/webpackage/go/bundle/cmd/gen-bundle@latest' to install the latest version